mysql.bom

```diff
@@ -0,0 +1,145 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+brooklyn.catalog:
+  version: "1.0.0-SNAPSHOT" # BROOKLYN_MYSQL_VERSION
+  publish:
+    description: |
+      MySQL is an open-source relational database management system (RDBMS).
+    license_code: Apache-2.0
+    defaults:
+      mysqlIconUrl: "https://www.mysql.com/common/logos/mysql-logo.svg"
+      classpathIconUrl: &mysqlIconUrl 'classpath:///mysql-logo-110x57.png'
+  items:
+  - "https://github.com/brooklyncentral/common-catalog-utils/releases/download/v0.1.0/common.bom"
+  - id: mysql
+    name: "MySQL"
+    description: |
+      MySQL is an open-source relational database management system (RDBMS).
+    itemType: entity
+    iconUrl: *mysqlIconUrl
+    item:
+      id: mysql
+      name: "MySQL"
+      type: centos7-software-process
+      brooklyn.parameters:
+      - name: mysql.port
+        label: "MySQL Port"
+        description: |
+          The MySQL port
+        type: integer
+        default: 3306
+      - name: mysql.datadir
+        label: "Data Directory"
+        description: |
+          The data directory to use
+        type: string
+      - name: datastore.creation.script.url
+        label: "Datastore Creation Script URL"
+        description: |
+          A URL to an SQL script to create the datastore
+        type: string
+      - name: mysql.password
+        label: "MySQL Root Password"
+        description: |
+          The root password to the MySQL server
+        type: string
+        default: s0M3r4nd0mPwd
```

Comment:
I think we try to avoid having default passwords now - force the user to always pick one